project-base/app/tests/FrontendApiBundle/Functional/Complaint/ComplaintTestTrait.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Complaint;
+
+use Shopsys\FrameworkBundle\Model\Complaint\Complaint;
+use Shopsys\FrameworkBundle\Model\Complaint\ComplaintItem;
+use Shopsys\FrontendApiBundle\Model\ScalarType\DateTimeType;
+
+trait ComplaintTestTrait
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Complaint\ComplaintItem $expectedComplaintItem
+     * @param array $complaintItem
+     */
+    protected function assertComplaintItem(ComplaintItem $expectedComplaintItem, array $complaintItem): void
```

Comment:
maybe `$actualComplaintItemData` is more descriptive?

Reply:
changed [fixup! get complaints query test](https://github.com/shopsys/shopsys/pull/3362/commits/0196828ec99ce8b131b88a9b1c47d83e90ca4a6f)